### PR TITLE
Fix latest experiment

### PR DESCRIPTION
**Pull request by So-G**
modified seed to create experiments in the past only

### CODE REVIEW

**Review by hkamalo — APPROVED** [review]
All good for me

File: models/ExperimentModel.js [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
 const { API_BASE_URL } = require('../env');
 
-// const findMany = (locationId, timestamp) =>
-//   prisma.experiment.findMany({
-//     where: { locationId: parseInt(locationId, 10), timestamp },
-//   });
-
-const findExperimentByTimestamp = async (locationId, timestamp) => {
-  const [response] = await prisma.experiment.findMany({
+const findExperimentByTimestamp = async (locationId, timestamp) =>
```
Comment (Herelius) [inline models/ExperimentModel.js:4]: You declared an async function but you did not use await in it.
you should add await after "=>" or delet the async statement.

File: models/ExperimentModel.js [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
+  prisma.experiment.findFirst({
     where: {
       locationId: parseInt(locationId, 10),
       timestamp,
     },
   });
-  return response;
-};
+
+const findLatestExperiment = async (locationId) =>
```
Comment (Herelius) [inline models/ExperimentModel.js:12]: You declared an async function but you did not use await in it.
you should add await after "=>" or delet the async statement.

**Review by dorianeGH — COMMENTED** [review]
working perfectly

**Pull request merged**